bigtop-tests/test-artifacts/hbase/src/main/groovy/org/apache/bigtop/itest/hbase/smoke/TestHBaseCompression.groovy

```diff
@@ -85,7 +85,7 @@ class TestHBaseCompression {
   }
 
   @Test
-  @Ignore("BIGTOP-218")
+  //@Ignore("BIGTOP-218")
```

Comment:
BIGTOP-218 is really very old(created on 2011). At that time snappy may not be included. And today bigtop already has libsnappy built-in, so this case can be enabled. And test passed on both x86 and aarch64.